hazelcast/src/main/java/com/hazelcast/instance/AbstractMember.java

```diff
@@ -16,15 +16,16 @@
 
 package com.hazelcast.instance;
 
-import com.hazelcast.nio.serialization.impl.BinaryInterface;
 import com.hazelcast.cluster.MemberAttributeOperationType;
 import com.hazelcast.core.Member;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.nio.Address;
```

Comment:
`Member` class in client protocol is serialized/deserialized using the dedicated `com.hazelcast.client.impl.protocol.codec.MemberCodec`, the serialized form implemented in `AbstractMember` is only relevant for member-to-member communication.